packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -705,6 +705,26 @@ const conf = (module.exports = convict({
       format: String,
     },
   },
+  surveys: {
+    default: [
+      // Example survey:
+      // {
+      //   id: 'portugese-speaking-mobile-users-in-southern-hemisphere',
+      //   conditions: {
+      //     platform: 'mobile',
+      //     region: 'southernHemisphere',
+      //     lang: 'pt',
+      //     relier: 'email',
+      //   },
+      //   view: 'settings',
+      //   rate: 0.1,
+      //   url: 'https://www.surveygizmo.com/s3/5541940/pizza',
+      // },
+    ],
+    doc: 'list of active surveys',
+    env: 'SURVEYS',
+    format: Array,
+  },
```

Comment:
@jrgm I think the idea to quickly deploy these surveys, without code changes, is through configs.  I just wanna get a 👍 from you to confirm a giant array like this as a config value is not a problem.

Reply:
(At this point I assume this will not be passed as a single env var; will ping jrgm for input.)

Reply:
So that's a list of `{id: .., conditions: .., ... }`? In yaml that would be like (abbreviated):
```
something:
  - conditions:
      lang: pt
    id: portugese-speaking-mobile-users-in-southern-hemisphere
    rate: 0.1
  - conditions:
      lang: pt
    id: portugese-speaking-mobile-users-in-southern-hemisphere
    rate: 0.1
```
Which isn't terribly far from https://github.com/mozilla-services/cloudops-deployment/blob/master/projects/fxa/puppet/yaml/type/fxa.auth_server.prod.yaml#L71-L75 (although hash not list). I probably can't preserve order in the list, if that matters to this application.

Reply:
> format: Array,

Ah, hmm. So node-convict considers `Array` to mean, "split this csv string on comma". - https://github.com/mozilla/node-convict/blob/master/packages/convict/src/main.js#L411-L412

So you probably want Object, even though it's an array, so that you get a JSON.parse of what is stuffed into the environment variable.

Reply:
(Or maybe my knowledge of convict is obsolete).

Reply:
@jrgm Thanks!  Yeah, I was concerned with the format (in addition to how you could set such a  big complex config, but that appears to be relatively simple in yaml) but wasn't sure if the config value would come thru as a single string.

@meandavejustice We need to change the config format to Object.